extensions/PhabBugz/lib/WebService.pm

```diff
@@ -34,6 +35,7 @@ use Bugzilla::Extension::PhabBugz::Util qw(
     request
 );
 
+use MIME::Base64 qw(decode_base64);
 use Data::Dumper;
```

Comment:
must've missed this before, but do we use Data::Dumper here or is this leftover from debugging?